src/shared/components/home/site-sidebar.tsx

```diff
@@ -23,12 +23,9 @@ import {
 
 interface SiteSidebarProps {
   site: Site;
-  localSite?: LocalSite;
-  admins?: PersonView[];
+  siteRes?: GetSiteResponse;
   isMobile?: boolean;
   myUserInfo: MyUserInfo | undefined;
-  allLanguages?: Language[];
-  siteLanguages?: number[];
```

Comment:
By removing all these its now confusing where you should use siteRes vs site. Just restore this.